### PR TITLE
test: add fuzzing case for the route with serverless-post-function

### PR DESCRIPTION
**Pull request by idbeta**
### What this PR does / why we need it:
relate #3340
fuzzing for the route with `post` and `serverless-post-function`

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by idbeta** [comment]
ping @spacewander @moonming @membphis

File: .github/workflows/fuzzing-ci.yaml [hunk 1]
```diff
@@ -51,9 +51,13 @@ jobs:
       run: |
         pip install typing boofuzz
 
-    - name: run test
+    - name: run simpleroute test
       run: |
-        python $PWD/t/fuzzing/route_test.py
+        python $PWD/t/fuzzing/simpleroute_test.py
+
+    - name: run serverless route test
```
Comment (spacewander) [inline .github/workflows/fuzzing-ci.yaml:58]: Why put it into separate step?

Comment (idbeta) [reply .github/workflows/fuzzing-ci.yaml:58]: I think this should be clearer. what do you think?

Comment (spacewander) [reply .github/workflows/fuzzing-ci.yaml:58]: I see.

**Comment by idbeta** [comment]
ping @membphis

**Review by membphis — APPROVED** [review]

**Pull request merged**